containers/Containerfile.ci

```diff
@@ -0,0 +1,77 @@
+FROM registry.fedoraproject.org/fedora:34
+
+MAINTAINER "Candlepin Team" <candlepin@redhat.com>
+# should be the UID of whatever user is running podman, change this with: podman build --build-arg UID="$(id -u)"
+ARG UID=1000
+ARG GIT_HASH=main
+ENV SMDEV_CONTAINER_OFF='True'
+
+RUN dnf install --disablerepo=updates -y \
+    cmake \
+    dbus-daemon \
+    dnf-utils \
+    gcc \
+    gettext \
+    git \
+    glibc-langpack-de \
+    glibc-langpack-ja \
+    intltool \
+    json-c-devel \
+    libdnf-devel \
+    libnotify-devel \
+    make \
+    npm \
+    openssl \
+    openssl-devel \
+    procps-ng \
+    python3-coverage \
+    python3-dateutil \
+    python3-decorator \
+    python3-devel \
+    python3-ethtool \
+    python3-gobject-base \
+    python3-iniparse \
+    python3-inotify \
+    python3-nose \
```

Comment:
nose is no more used, so this should be replaced with:
- python3-pytest
- python3-pytest-flake8
- python3-pytest-forked
- python3-pytest-randomly
- python3-pytest-timeout